ftw/testbrowser/drivers/mechdriver.py

```diff
@@ -59,6 +60,11 @@ def make_request(self, method, url, data=None, headers=None,
         try:
             self.response = self._get_mechbrowser().open(request)
         except urllib2.HTTPError as response:
+            if response.reason.startswith(
+                    'The HTTP server returned a redirect error that would'
```

Comment:
repeating and relying on that exact message looks a bit brittle, could you import the message from mechanize instead (`HTTPRedirectHandler.info_msg`) ?